### PR TITLE
feat: schema for Subwayside

### PR DESCRIPTION
**Pull request by paulswartz**
You can see the live version at https://paulswartz.github.io/schemas/events/com.mbta.ctd.subwayside.train-heartbeat

### CODE REVIEW

File: schemas/com.mbta.ctd.subwayside.train-heartbeat.json [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+    "carNumber": {
+      "type": "string",
+      "description": "The 4 digit number of a car",
+      "pattern": "^\\d{4}$"
+    },
+    "timestamp": {
+      "description": "RFC3339 timestamp",
+      "type": "string",
+      "format": "date-time",
+      "pattern": "^\\d{4}-(0\\d|1[012])-([012]\\d|3[01])T([01]\\d|2[0-3]):([0-5]\\d):([0-5]\\d)([.]\\d{1,6})?(Z|[+-]\\d{1,2}:[0-5]\\d)$"
```
Comment (beatrixada) [inline schemas/com.mbta.ctd.subwayside.train-heartbeat.json:56]: question: is this validation pattern necessary? apparently JSON Schema already expects RFC 3339 datetimes: https://json-schema.org/understanding-json-schema/reference/string.html#dates-and-times

**Review by beatrixada — APPROVED** [review]
@paulswartz I notice most of the fields aren't documented. I don't personally care about that because it sounds like we really only need three of them for crowding, but do we plan to document them more extensively somewhere?

File: schemas/com.mbta.ctd.subwayside.train-heartbeat.json (hunk 1 above)
Comment (paulswartz) [reply schemas/com.mbta.ctd.subwayside.train-heartbeat.json:56]: `format` isn't required to be implemented by validators. From slightly above your link:
> By default, `format` is just an annotation and does not effect validation.

**Comment by beatrixada** [comment]
Oh one last question: is there a specific reason so many fields are named `*Dt` etc instead of `*Date`?

**Review by beatrixada — COMMENTED** [review]
question: why do some fields use abbreviations like `nbr` or `Dt`?

**Comment by paulswartz** [comment]
> Oh one last question: is there a specific reason so many fields are named *Dt etc instead of *Date?

`*Dt` is for date-time. I'd probably have used `*Date` for actual dates, or `*Time` for times.

> question: why do some fields use abbreviations like nbr or Dt?

The fields are `nbr` or `*nbr` in the database, but I can change that. Some options that come to mind:
- `number` 
- `carNumber`
- `carId`

**Comment by paulswartz** [comment]
@bklebe I've updated all the `Nbr` to `Number`, but kept `Dt`.

**Pull request merged**